tests/integration/it_owner_privilages_test.py

```diff
@@ -1,25 +1,35 @@
+""" Tests for privilage of user.
+"""
+
 from datetime import datetime
 
 import flask
-
+import pytest
 from timeless.access_control.methods import Method
 from timeless.access_control.owner_privileges import has_privilege
 from timeless.employees.models import Employee
 
 
-def test_can_access_location(app):
+@pytest.fixture
+def clean_app(app):
+    """ Fixture for cleaning global variables in tests. """
+    user = getattr(flask.g, "user", None)
+    if user is not None:
+        flask.g.user = None
```

Comment:
@amihaiemil in this test file there was only the user attribute.

Reply:
@ammaratef45 I look again now. Does it make sense to read the ``user``, check if it's not ``None`` and then set it to ``None``? We can directly say ``flask.g.user = None``, it would be the same thing, right?

Reply:
@amihaiemil You're right :+1: